tests/dragonfly/cluster_test.py

```diff
@@ -2256,10 +2256,13 @@ async def test_migration_timeout_on_sync(df_factory: DflyInstanceFactory, df_see
     await nodes[1].client.execute_command("debug migration resume")
 
     await asyncio.sleep(1)  # migration will start resync
-    seeder.stop()
-    await seeder_task
 
     await wait_for_status(nodes[0].admin_client, nodes[1].id, "FINISHED")
 
-    capture = await seeder.capture()
+    nodes[0].migrations = []
+    nodes[0].slots = []
+    nodes[1].slots = [(0, 16383)]
+
+    await push_config(json.dumps(generate_config(nodes)), [node.admin_client for node in nodes])
```

Comment:
what happens if you check the capture before you push the config? shouldnt this work as well?

Reply:
it works, but it uses a cluster client and can do redirections that makes it a little unstable